tagilmo/utils/vereya_wrapper.py

```diff
@@ -466,7 +472,12 @@ def clear(self):
 
     def getCachedObserve(self, method, key = None):
         with self.lock:
-            val = self.cached[method][0]
+            val = self.cached[method]
+        if method in self.events and val is not None:
+            self.readEvents[method] = True
+            self.cached[method] = [(None, 0)]
+        else:
+            val = val[0]
```

Comment:
This will fail if val is None

Reply:
val = val[0] will fail or what? Why should it be None? Look, here how it was:
```
with self.lock:
    val = self.cached[method][0]
```

And here how it looks now:
```
with self.lock:
    val = self.cached[method]
...
val = val[0]
```

What is the difference?

Reply:
Agree. I've deleted it.